dvc/remote/gs.py

```diff
@@ -17,13 +17,13 @@
 from dvc.utils.compat import FileNotFoundError  # skipcq: PYL-W0622
 
 logger = logging.getLogger(__name__)
+MIN_CHUNKSIZE = 256 * 1024
```

Comment:
Let's use Blob._CHUNK_SIZE_MULTIPLE as we did before, to keep the reference for the internals.

Reply:
I was hesitant as the `_` prefix implies it's not public API.

Reply:
I was too :( But at least we were tied to an actual source. Also we have google-cloud-storage version fixed to one specific version, so if that value disappears, we'll notice it right away.